benchmarks/6.1.0/v1/README.md

```diff
@@ -0,0 +1,14 @@
+# V1
+
+In this version of Performance results the tests were executed with the following configs added with the intension of disabling the newly introduced features/improvements. This has been done in order to ease cross-comparison with earlier versions of the product.
```

Comment:
In this version of Performance results, the tests were executed with the following configs added which disable some of the newly added features/improvements. This has been done in order to ease cross-comparison with earlier versions of the product.

`add config details`

In order to see the performance results with the with above features enabled, please refer [the v2 results](url) which use the default pack without enabling or disabling any additional features.

Please note that in both cases, [performance tuning parameters](https://is.docs.wso2.com/en/latest/deploy/performance/performance-tuning-recommendations/) has been fine tuned as needed.

Reply:
Fixed with 9ca8e3c41cafd7135c9c35f0940fb8c9e9a35d68